site/webpack.config.js

```diff
@@ -0,0 +1,68 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+// directory where compiled assets will be stored
+    .setOutputPath('web/build/')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
+    // only needed for CDN's or sub-directory deploy
+    //.setManifestKeyPrefix('build/')
+
+    /*
+     * ENTRY CONFIG
+     *
+     * Add 1 entry for each "page" of your app
+     * (including one that's included on every page - e.g. "app")
+     *
+     * Each entry will result in one JavaScript file (e.g. app.js)
+     * and one CSS file (e.g. app.css) if you JavaScript imports CSS.
+     */
+    //css entries
+    .addStyleEntry('app-css', ['./web/assets/scss/librecores.scss',
+        './web/assets/scss/bootstrap-librecores.scss',
+        './web/assets/css/bootstrap-social.css',
+        './web/assets/css/font-awesome.min.css',
+        './web/assets/css/librecores.css'])
+    .addStyleEntry('trumbowyg', ['./web/assets/scss/trumbowyg.scss'])
+    .addStyleEntry('project_new_css','./web/assets/css/bootstrap-select.css')
+
+    //js entries
+    .addEntry('app-js',['./web/assets/js/livestamp.js'])
+
+    //page specific js
+    .addEntry('search', ['./web/assets/js/search.js'])
+    .addEntry('project-view', ['./web/assets/js/metrics.js',
+        './web/assets/js/project-auto-refresh.js',
+        './web/assets/css/chartist.css'])
+    .addEntry('project_settings', './web/assets/js/classification.js')
+    .addEntry('project_new', ['./web/assets/js/bootstrap-select.js',
+        './web/assets/js/project_new.js'])
+    .addEntry('planet', './web/assets/js/planet.js')
+    .addEntry('organisation', './web/assets/js/organisation.js')
```

Comment:
Please make these file names consistently using the page name.

Reply:
Knew that naming conventions would be a problem. What do you recommend? 
Planet - `planet_page` , Organisation: `organisation_page`?

Reply:
Ideally you find a convention that matches the URL fragment and/or the html template (in views). So most of them are fine most likely, except for `project-view`.

On the CSS side, project_new_css and trumbowyg have inconsistent names.

And please think about `app-js` and `app-css`. Do you need the js/css suffixes? And can we keep things consistent by either using underscores or dashes?

Reply:
Sure thing, We can use `base.css` `base.js` as they are in `base.html.twig`